src/manager/frontend/src/client/util.ts

```diff
@@ -0,0 +1,3 @@
+export function getBaseURL(): string {
+  return typeof window !== "undefined" ? window.location.href : "";
```

Comment:
This is working but a bit crunchy, we have it in RAGapp too (using `window.location.origin`) which is broken with reverse proxy

Reply:
using a deployment variable and if not set fallback to window.location?